packages/react-router-dom-v5-compat/rollup.config.js

```diff
@@ -86,7 +86,7 @@ module.exports = function rollup() {
         banner: createBanner("React Router DOM v5 Compat", version),
         globals: {
           history: "HistoryLibrary",
-          "@remix-run/router": "Router",
+          "@remix-run/router": "RemixRouter",
```

Comment:
The UMD name for the build is `RemixRouter` for clarity - `Router` felt a bit vague 😄